R/classicProcess.R

```diff
@@ -76,8 +76,11 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
 # Helper function for generic dependencies of all models
 .procGetDependencies <- function() {
   return(c(
-    "dependent", "covariates", "factors", "naAction", "emulation", "estimator", "meanCenteredModeration",
-    "standardizedModelEstimates", "errorCalculationMethod", "bootstrapCiType"
+    "dependent", "covariates", "factors", "naAction", "emulation", "estimator",
+    "meanCenteredModeration", "standardizedModelEstimates", "errorCalculationMethod", "bootstrapCiType",
+    "mcmcBurnin", "mcmcSamples", "mcmcChains", "seed", "setSeed", "nuPriorMu",
+    "nuPriorSigma", "betaPriorMu", "betaPriorSigma", "psiPriorAlpha",
+    "psiPriorBeta", "rhoPriorAlpha", "rhoPriorBeta"
```

Comment:
They run independently, but `BayesianProcess` reuses many functions from `ClassicProcess` (including the GUI dependencies), so it was much easier to add the Bayesian-specific GUI dependencies there. In theory, you are right, it would be better to have them in a separate function, but then we would need to adapt many of the existing functions (which I didn't do in the interest of time).